.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Report unexpected behavior or any issues you experienced in OpenRA.
+title: ''
+labels: bug
```

Comment:
OpenRA uses `bug` with capital `B`. Can change it when the testcase is not needed anymore.